### PR TITLE
Dont include grant type

### PR DESCRIPTION
**Pull request by lizgzil**
As @antonio pointed out in https://github.com/wellcometrust/nutrition-labels/issues/62
"I think using grant types is a bit problematic giving the new strategy - they are probably going to change, so using something that only uses text or fields that won't change is preferable?"

Thus I included not using grant types in the experiments, and after finding out performance isn't drastically effected, I created a new config file for this case. Results in the documents are updated.

I also reran the ensemble parameter experiments with these models and include several new images for the doc.

(Next PR: use these ensemble params in a config, and update ensemble class to use wellcomeml voting classifier.)

### CODE REVIEW

**Comment by lizgzil** [comment]
[Running models without grant types text in training data](https://trello.com/c/0xjgV2Aj/1219-running-models-without-grant-types-text-in-training-data)

**Comment by lizgzil** [comment]
I also updated WellcomeML to the latest commit. This included the fixes to the bert vectorizer made in this [PR](https://github.com/wellcometrust/WellcomeML/pull/182/files).

This change means that old bert/scibert vectorizers can no longer be loaded - The latest results from the `210401` models are the first to be compatible with the new wellcomeml.

**Comment by lizgzil** [comment]
momentous time @nonieA - recall and precision are at 0.9!!

**Comment by nonieA** [comment]
AHHHH amazing 🙂
________________________________

File: docs/Tech_Grants_2021.md [hunk 1]
```diff
@@ -93,36 +95,71 @@ Thus `grant_tagger.py` was adapted to:
...
+
+This ensemble gives the following results on the test set:
+
+||precision|recall |f1-score   |support|
+|--|---|---|---|---|
+|Not tech|0.89|0.91|0.90|77|
+|Tech|0.91|0.89|0.90|80|
+|accuracy|||0.90|157|
+|macro avg|0.90|0.90|0.90|157|
+|weighted avg|0.90|0.90|0.90|157|
```
Comment (aCampello) [inline docs/Tech_Grants_2021.md:160]: So we got to 0.9, the holy grail?

File: nutrition_labels/grant_tagger_evaluation.py [hunk 2]
```diff
@@ -108,12 +108,12 @@ def get_model_scores_df(config, args):
...
-    try:
-        model_names.remove('test_data_formatted.csv')
-    except:
-        print('test_data_formatted.csv not created yet')
+
+    # Remove items in this directory which aren't model names
+    not_model_names = ['training_information.json', '.DS_Store', 'test_data_formatted.csv']
+    for not_model_name in not_model_names:
+        if not_model_name in model_names:
+            model_names.remove(not_model_name)
```
Comment (aCampello) [inline nutrition_labels/grant_tagger_evaluation.py:116]: list comprehension would be more concise? `model_names = [model for model in model_name is model not in not_model_names]`?

Also, I feel like a "positive" logic would be more readable, i.e. what are the files that are actually models? (`.ini,?`)

**Review by aCampello — APPROVED** [review]
LGTM - Small comments if you want to address.

**Review by aCampello — APPROVED** [review]

File: nutrition_labels/grant_tagger_evaluation.py (hunk 2 above)
Comment (lizgzil) [reply nutrition_labels/grant_tagger_evaluation.py:116]: yep very good point!

File: docs/Tech_Grants_2021.md (hunk 1 above)
Comment (lizgzil) [reply docs/Tech_Grants_2021.md:160]: finally!

**Pull request merged**